jazzygrid/patterns/home.php

```diff
@@ -0,0 +1,153 @@
+<div class="wp-block-group" style="margin-top:0px;margin-bottom:0px;padding-top:0px;padding-right:0px;padding-bottom:0px;padding-left:0px"><!-- wp:heading {"level":1} -->
+<h1 class="wp-block-heading"><?php echo __('<a rel="noreferrer noopener" href="https://www.wordpress.org/" target="_blank">Message in Jaz</a><a href="https://www.wordpress.org/" target="_blank" rel="noreferrer noopener">z</a>', 'jazzygrid');?></h1>
+<!-- /wp:heading -->
+
+<!-- wp:paragraph -->
+<p><?php echo __('TJM brings trumpeter Maxine Davis, saxophonist D. James, Tiana Alvarez on the piano, Derrick Hayes on bass, and Marcus Lee Jones on the drums. Together, they create an engaging experience to delight fans of classic and contemporary jazz.', 'jazzygrid');?></p>
+<!-- /wp:paragraph --></div>
+<!-- /wp:group -->
+
+<!-- wp:group {"layout":{"type":"flex","orientation":"vertical","justifyContent":"stretch"}} -->
+<div class="wp-block-group"><!-- wp:paragraph {"style":{"typography":{"textTransform":"uppercase","fontSize":"0.76rem","letterSpacing":"0.1rem"}}} -->
+<p style="font-size:0.76rem;letter-spacing:0.1rem;text-transform:uppercase"><?php echo __('<a href="#">Coming Next</a>', 'jazzygrid');?></p>
+<!-- /wp:paragraph -->
+
+<!-- wp:paragraph -->
+<p><?php echo __('Live at the Masonic Auditorium','jazzygrid'); ?><br><?php echo __('San Francisco, July 22nd, 2023.','jazzygrid'); ?></p>
+<!-- /wp:paragraph --></div>
+<!-- /wp:group --></div>
+<!-- /wp:group --></div></div>
+<!-- /wp:cover --></div>
+<!-- /wp:column -->
+
+<!-- wp:column {"verticalAlignment":"stretch","width":"25%","backgroundColor":"secondary"} -->
+<div class="wp-block-column is-vertically-aligned-stretch has-secondary-background-color has-background" style="flex-basis:25%"><!-- wp:group {"style":{"spacing":{"margin":{"top":"0rem","bottom":"0rem"},"padding":{"top":"4rem","right":"3rem","bottom":"4rem","left":"1.5rem"},"blockGap":"2.5rem"},"dimensions":{"minHeight":"100%"}},"layout":{"type":"flex","orientation":"vertical","verticalAlignment":"space-between"}} -->
+<div class="wp-block-group" style="min-height:100%;margin-top:0rem;margin-bottom:0rem;padding-top:4rem;padding-right:3rem;padding-bottom:4rem;padding-left:1.5rem"><!-- wp:group {"style":{"spacing":{"padding":{"top":"0px","right":"0px","bottom":"0px","left":"0px"},"margin":{"top":"0px","bottom":"0px"},"blockGap":"1.5rem"},"elements":{"link":{"color":{"text":"var:preset|color|foreground"}}}},"textColor":"foreground","layout":{"type":"flex","orientation":"vertical"}} -->
+<div class="wp-block-group has-foreground-color has-text-color has-link-color" style="margin-top:0px;margin-bottom:0px;padding-top:0px;padding-right:0px;padding-bottom:0px;padding-left:0px"><!-- wp:heading {"level":1} -->
+<h1 class="wp-block-heading"><?php echo __('<a href="https://en.wikipedia.org/wiki/Billie_Holiday" target="_blank" rel="noreferrer noopener"></a><a href="#" target="_blank" rel="noreferrer noopener">Jazzin\' Around Town</a>', 'jazzygrid');?></h1>
```

Comment:
Seems like there's an empty `anchor` and then a used one, should these be just one anchor? Not sure if that's the correct URL for the text though

```suggestion
<h1 class="wp-block-heading"><?php echo __('<a href="https://en.wikipedia.org/wiki/Billie_Holiday" target="_blank" rel="noreferrer noopener">Jazzin\' Around Town</a>', 'jazzygrid');?></h1>
```